patches/server/0004-Paper-config-files.patch

```diff
@@ -148,6 +148,7 @@ index 0000000000000000000000000000000000000000..31325994ab441c59a4c0bd9f3f9db3d9
 +
 +public abstract class Configurations<G, W> {
 +
++    private static final org.slf4j.Logger LOGGER = com.mojang.logging.LogUtils.getLogger();
```

Comment:
You can use imports here, as this patch is the patch that creates the file. So go ahead and import LogUtils and Logger and use those directly.

We like to avoid adding imports to files that we are changing as that can create useless conflicts on rebases. But if the file is being first created, you can add as many imports as you want as its never going to conflict on a rebase.

Reply:
Done